rosetta/cosmos/launchpad/network.go

```diff
@@ -17,8 +23,45 @@ func (l Launchpad) NetworkList(ctx context.Context, request *types.MetadataReque
 	}, nil
 }
 
+type nodeInfo struct {
+	Version string `json:"version"`
+}
+
+type nodeResponse struct {
+	NodeInfo nodeInfo `json:"node_info"`
+}
+
 func (l Launchpad) NetworkOptions(ctx context.Context, request *types.NetworkRequest) (*types.NetworkOptionsResponse, *types.Error) {
-	panic("implement me")
+	addr := l.endpoint + endpointNodeInfo
+	resp, err := l.c.Get(addr)
+	if err != nil {
+		return nil, rosetta.NewError(1, "error getting data from node")
```

Comment:
This is a good moment, as you did with the other to put all the errors in the same place. I like it and is easy to see which errors our app does.

Reply:
done by 823e030